saleor/settings.py

```diff
@@ -89,6 +89,8 @@ def get_bool_from_env(name, default_value):
     # ),
 }
 
+DATABASE_ROUTERS = ["saleor.core.db_routers.PrimaryReplicaRouter"]
```

Comment:
We'll have to override this class in cloud builds, however methods used here does not conflict with the DB router for cloud.

Reply:
What Saleor versions will have this change?

Reply:
Right now this goes only to main, so 3.8 onwards.